service.go

```diff
@@ -39,9 +39,14 @@ type CreateServiceOptions struct {
 // or an error in case of failure.
 //
 // See https://goo.gl/KrVjHz for more details.
-func (c *Client) CreateService(opts CreateServiceOptions) (*swarm.Service, error) {
+func (c *Client) CreateService(opts CreateServiceOptions, auth AuthConfiguration) (*swarm.Service, error) {
```

Comment:
Can you add AuthConfigure to the CreateServiceOptions struct and set it to `qs:"-"`? This way we don't break the API.